packages/vite/src/node/plugins/css.ts

```diff
@@ -2183,6 +2183,10 @@ async function compileLightningCSS(
             if (filePath === filename) {
               return src
             }
+            // This happens with html-proxy (#13776)
+            if (!filePath.endsWith('.css')) {
+              return src
+            }
```

Comment:
When running the playground, I got `""` (empty string) as a value of `filePath`. I think this doesn't make sense and it's a bug in Lightning CSS. But I think it's fine to merge this as a workaround for now.